_health-care/_js/reducers/veteran/index.js

```diff
@@ -13,171 +13,171 @@ _.mixin(lodashDeep);
 const blankVeteran = {
   nameAndGeneralInformation: {
     fullName: {
-      first: null,
-      middle: null,
-      last: null,
-      suffix: null,
+      first: makeField(''),
+      middle: makeField(''),
+      last: makeField(''),
+      suffix: makeField(''),
     },
-    mothersMaidenName: null,
-    socialSecurityNumber: null,
+    mothersMaidenName: makeField(''),
+    socialSecurityNumber: makeField(''),
     dateOfBirth: {
-      month: null,
-      day: null,
-      year: null,
+      month: makeField(''),
+      day: makeField(''),
+      year: makeField(''),
     },
-    maritalStatus: null,
-    sectionComplete: false
+    maritalStatus: makeField(''),
+    sectionComplete: makeField(false)
```

Comment:
We might not want to change `sectionComplete` from just being `false`, since this is currently breaking form styles.